server/controllers/memoCtrl.js

```diff
@@ -444,9 +445,15 @@ async function getAboutContent(req, res, next) {
     webContext.infoContactName = infoContactName
     webContext.examiners = examiners
 
-    webContext.memoDatas = await markOutdatedMemoDatas(rawMemos, roundInfos)
+    const memoDatas = await markOutdatedMemoDatas(rawMemos, roundInfos)
```

Comment:
It is better to be `webContext.memoDatas = await markOutdatedMemoDatas(rawMemos, roundInfos)` two lines are replaced by one, no need in one extra variable

Reply:
done